### PR TITLE
Move istio-install to one yaml file

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

File: kubernetes/istio.yaml [hunk 1]
```diff
@@ -153,3 +204,73 @@ spec:
...
+              apiVersion: v1
+              fieldPath: metadata.namespace
+---
+################################
+# Istio CA
+################################
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: istio-ca
```
Comment (myidpt) [inline kubernetes/istio.yaml]: Can we keep this as a separate file? Users will want flexibility whether deploying CA, and in which namespace to deploy CA.

Comment (andraxylia) [inline kubernetes/istio.yaml]: Yes, we can.

Comment (myidpt) [inline kubernetes/istio.yaml]: Thanks!

**Comment by frankbu** [comment]
We need to merge this ASAP because the istio install doc is now broken until this PR is merged. This change breaks the kubeTest.sh, which needs to be fixed. The following function in tests/commonUtils.sh needs to be changed to work with the single yaml file:
```
function generate_istio_yaml() {
    print_block_echo "Generating istio yaml in ${1}"
    local src_dir="${ROOT}/kubernetes/istio-install"
    local dest_dir="${1}"

    mkdir -p ${dest_dir}
    cp ${src_dir}/* ${dest_dir}
    sed -i "s|image: .*/\(.*\):.*|image: $MANAGER_HUB/\1:$MANAGER_TAG|" ${dest_dir}/istio-manager.yaml
    sed -i "s|image: .*/\(.*\):.*|image: $MANAGER_HUB/\1:$MANAGER_TAG|" ${dest_dir}/istio-ingress-controller.yaml
    sed -i "s|image: .*/\(.*\):.*|image: $MIXER_HUB/\1:$MIXER_TAG|" ${dest_dir}/istio-mixer.yaml
}
```

**Comment by andraxylia** [comment]
I close this PR, same change + support for RBAC enabled clusters will be merged soon with #150

**Pull request closed without merging**